.github/workflows/build.yml

```
@@ -147,12 +147,41 @@
 
+      - name: Generate auto-updater metadata
+        run: |
+          # Generate latest-mac.yml for auto-updater
+          ZIP_FILE=$(ls dist/*.zip | head -1)
+          ZIP_NAME=$(basename "$ZIP_FILE")
+          ZIP_SIZE=$(stat -f%z "$ZIP_FILE")
+          ZIP_SHA512=$(shasum -a 512 "$ZIP_FILE" | cut -d' ' -f1)
+          BLOCKMAP_FILE="${ZIP_FILE}.blockmap"
+          BLOCKMAP_SIZE=$(stat -f%z "$BLOCKMAP_FILE")
+          BLOCKMAP_SHA512=$(shasum -a 512 "$BLOCKMAP_FILE" | cut -d' ' -f1)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)